lib/record.gd

```diff
@@ -128,20 +128,32 @@ DeclareOperationKernel( "Unbind.", [ IsObject, IsObject ], UNB_REC );
 ##  <Attr Name="RecNames" Arg='record'/>
 ##
 ##  <Description>
-##  returns a list of strings corresponding to the names of the record
-##  components of the record <A>record</A>.
+##  returns a duplicate-free list of strings corresponding to the names of
+##  the record components of the record <A>record</A>.
+##  <P/>
+##  The list is sorted by <Ref Func="RNamObj" Label="for a string"/>
+##  for reasons of efficiency; see <Ref Oper="SortBy"/>.
+##  Therefore the ordering is consistent with one &GAP; session,
+##  but it is not necessarily consistent across different sessions.
+##  One could apply, for instance, <Ref Oper="Set"/> to obtain a result that
+##  is consistent across &GAP; sessions.
+##  <P/>
+##  Note that you cannot use the string result in the ordinary way to access
+##  or change a record component.
+##  You can use the <C><A>record</A>.(<A>name</A>)</C> construct for that,
+##  see Sections&nbsp;<Ref Sect="Accessing Record Elements"/>
+##  and&nbsp;<Ref Sect="Record Assignment"/>.
```

Comment:
This also incorrectly formats `name` as an argument.

Perhaps this would be better:
```suggestion
##  Note that given a string <C>name</C> containing the name of a record component,
##  you can access the record component via <C><A>record</A>.(<A>name</A>)</C>,
##  see Sections&nbsp;<Ref Sect="Accessing Record Elements"/>
##  and&nbsp;<Ref Sect="Record Assignment"/>.
```

Reply:
Thanks for the review and this suggestion @fingolfin. It seems sensible at first glance. I’ll get back to this PR in a few days 🙂